plugins/sdm/accessbot.py

```diff
@@ -126,6 +158,7 @@ def access_resource(self, message, match):
         """
         Grant access to a resource (using the requester's email address)
         """
+        self.increment_metrics([MetricGaugeType.TOTAL_RECEIVED_MESSAGES, MetricGaugeType.TOTAL_ACCESS_REQUESTS])
```

Comment:
Do (for example):
```
self.metrics_helper.increase_access_requests()
```

Instead of:
```
self.increment_metrics([MetricGaugeType.TOTAL_RECEIVED_MESSAGES, MetricGaugeType.TOTAL_ACCESS_REQUESTS])
```